src/verilog.y

```diff
@@ -1718,11 +1744,14 @@ parameter_port_declarationTypeFrontE: // IEEE: parameter_port_declaration w/o as
         ;
 
 net_declaration<nodep>:         // IEEE: net_declaration - excluding implict
-                net_declarationFront netSigList ';'     { $$ = $2; }
+                net_declarationFront netSigList ';'     { $$ = $2;
+    if (GRAMMARP->m_netStrengthp) delete GRAMMARP->m_netStrengthp; }
```

Comment:
```suggestion
                net_declarationFront netSigList ';'
                        { $$ = $2;
                          if (GRAMMARP->m_netStrengthp) {
                              VL_DO_CLEAR(delete GRAMMARP->m_netStrengthp, GRAMMARP->m_netStrengthp = nullptr);
                          }}
```

Otherwise m_netStrengthp is a dangling pointer.